boardswarm-client/src/authenticator.rs

```diff
@@ -86,7 +86,7 @@ where
             if let Some(token) = auth.token().await {
                 req.headers_mut().insert(
                     http::header::AUTHORIZATION.as_str(),
-                    format!("Bearer {}", token).parse().unwrap(),
+                    format!("Bearer {}", token.trim()).parse().unwrap(),
```

Comment:
The other option is to trim when `Authenticator::from_static()` is called.

Reply:
Yeah better to do it there rather then forcing the users to clean up; I do wonder how you got that in your config

Reply:
For whatever reason, when using the auth/configure init flow to pull from a regular file, I ended up with:
```yaml
auth: !Token |
    foo
```

I'm not sure why `serde_yaml` chose to emit block style, given that when reading from a token file we right-strip it on entry. I equally have no idea what the most idiomatic way to right-strip it when reading from YAML would be ...

Reply:
(Using `|-` to specify right-chomp on deserialisation works, but I couldn't see how to make `serde-yaml` emit that on serialisation, and it's also deprecated ...)

Reply:
Odd for me it always serializes with `!Token foo` rather then doing a block ; And yeah serde-yaml got inactive really, i've been waiting to see what replacement really takes of before switching

Reply:
fwiw Given the confusion with `configure` (which is older) vs `auth` makes me wonder if you ended up running a somewhat old git version of boardswarm before 7baa91bb at some point triggering this